lib/Importer/ONNX.cpp

```diff
@@ -84,19 +84,6 @@ std::vector<size_t> getPads(const ArgumentDictionaryTy &dict) {
   return {0, 0, 0, 0};
 }
 
-size_t getPad(const ArgumentDictionaryTy &dict) {
-  if (dict.count("pads")) {
-    return getConstantArrayHead(dict.at("pads"));
-  }
-  if (dict.count("auto_pad")) {
-    auto padStr = loadStr(dict.at("auto_pad"));
-    if (padStr == "VALID")
```

Comment:
Is this stuff about auto_pad and VALID relevant anymore?  It looks like we lost some functionality here.

Reply:
Yes, in function getPads(). This "getPad" function was used to only support equal-padding. Now since non-square padding is supported, this one is replaced by "getPads", which wad added in conv non-square padding PR already.

Reply:
Ah, got it, thanks!  I was confused by the getPads() above in the Caffe2 importer :)